src/.buildkite/test/docker-compose.yml

```diff
@@ -7,9 +7,12 @@ services:
     working_dir: /app
     command: /app/src/.buildkite/test/run.sh
     environment:
+      - TEST_POSTGRES_BASE_URI=postgres://prisma:prisma@postgres:5432
```

Comment:
These were added to support new integration test system that wants to control what the database will be.